kurobako_problems/src/zdt.rs

```diff
@@ -54,7 +54,7 @@ impl ProblemFactory for ZdtProblemFactory {
                 "paper",
                 "Zitzler, Eckart, Kalyanmoy Deb, and Lothar Thiele. \"Comparison of multiobjective \
                  evolutionary algorithms: Empirical results.\" Evolutionary computation 8.2 (2000): 173-195."
-            ).value(domain::var("f1")).value(domain::var("f2"));
+            ).value(domain::var("f1")).value(domain::var("f2")).reference_point(Some(Params::new(vec![11.0, 11.0])));
```

Comment:
Just out of curiosity. Could you tell me where the reference point `[11.0, 11.0]` came from?

Reply:
It's based on [BoTorch's test set](https://github.com/pytorch/botorch/blob/master/botorch/test_functions/multi_objective.py). Might be worth noting that BoTorch does not provide the full set of ZDT functions as Kurobako does. I observed that we could get strange hypervolumes with e.g. ZDT4.

Reply:
Got it. Thanks!